tests/nat/middleware/test_defense_middleware.py

```
@@ -0,0 +1,655 @@
+
+        # Extract field from output if target_field is specified
+        content, field_info = self._extract_field_from_value(output)
+
+        # Store extracted content for test verification
+        self._last_extracted_content = content
+        self._last_field_info = field_info
+
+        # Return output
+        return output
+
+    async def function_middleware_stream(self, value, call_next, context):
```

Ruff (0.14.8)

Unused method argument: `context`

(ARG002)